variables.tf

```diff
@@ -25,3 +25,13 @@ variable "nodes_name" {
   default     = "nodes"
   description = "K8s nodes subdomain name in the Kops DNS zone"
 }
+
+variable "vpc_tag" {
+  default     = "Name"
+  description = "Use Name tag to lookup the Kops VPC"
```

Comment:
This description is now misleading. It should probably say "Tag used to lookup the Kops VPC"